articles/annotation_and_query.md

```diff
@@ -141,149 +131,143 @@ To enter multiple Values for a single Key click `Enter` with the cursor in the V
+```
+SELECT * FROM syn123456 WHERE "fileFormat"='fastq'
+```
 
-Along with annotations added by users, every entity has a number of fields useful for searching. For a complete list, see [here](http://hud.rel.rest.doc.sagebase.org.s3-website-us-east-1.amazonaws.com/org/sagebionetworks/repo/model/Entity.html). 
+```
+SELECT * FROM syn123456 WHERE "RIN"<=6.1
+```
 
-Here are some really useful ones:
+Along with annotations added by users, every entity has a number of fields useful for searching. For a complete list, see:
 
-* `projectId` which project the entity is associated with
-* `parentId`: where the entity resides (inside a folder/sub-folder; may also be the project if in the root folder) - useful for finding all files if you know they are in a specific folder
 
+<a href="https://docs.synapse.org/rest/org/sagebionetworks/repo/web/controller/TableExamples.html" target="_blank" class="btn btn-info btn-lg active" role="button" aria-pressed="true">SQL Query Examples</a>
 
 
 <br/>
 
-### Finding files in a specific project
+### Finding Files in a Specific Project
 
-To find all files in a specific `Project`, you need to know the `projectId`, which is a Synapse identifier (looks like `syn12345`).
-For example, the TCGA Pan-Cancer Project has a `projectId` of syn300013.
-So, the query to find all files and all annotations associated with this `Project` would be:
-
-```
-SELECT * FROM file WHERE projectId=="syn300013"
-```
+To find all files in a specific `Project`, create a `File View` in the web client. For example, if you'd like to see all files in a `Project`, navigate to your project and then the `Tables` tab. From there, click **Tables Tools** and **Add File View**. Click **Add container** and **Enter Synapse Id** to create a tabluar file view that contains every file in the project, which you can now query. Importantly, if you want to later query on annotations, you must select **Add All Annotations**. For a more in-depth look at this feature, please read our articles on [File Views](/articles/views.html).
 
 
 <br/>
 
-### Listing files in a specific folder
+### Listing Files in a Specific Folder
 
-To list the `Files` and annotations in a specific `Folder`, you need to know the Synapse ID of the `Folder` (for example syn1524884, which has data from TCGA related to melanoma). All entities in this `Folder` will have a `parentId` of syn1524884.
+To list the `Files` in a specific `Folder`, you need to know the synID of the `Folder` (for example syn1524884, which has data from TCGA related to melanoma). All entities in this `Folder` will have a `parentId` of syn1524884.
 
-So, the query to find all `Files` and all annotations in this `Folder` would be:
+The function to find all `Files` in this `Folder` is called "getChildren":
```

Comment:
note for @meredithslota @kimyen - we should document this as a `list` function (`synList`; that @kimyen was probably porting from the command line client).